packages/react-native/ReactCommon/react/renderer/components/view/conversions.h

```diff
@@ -513,48 +515,52 @@ inline void fromRawValue(
         transformMatrix.matrix[i++] = number;
       }
       transformMatrix.operations.push_back(
-          TransformOperation{TransformOperationType::Arbitrary, 0, 0, 0});
+          TransformOperation{TransformOperationType::Arbitrary, Zero, Zero, Zero});
     } else if (operation == "perspective") {
```

Comment:
I missed on my initial look-through that we are using `ValueUnit` (`Point`, or `Percentage`) for operations which take radians, which is a little confusing.

Don't want to make this PR too big, but we would replace `ValueUnit` will `CSSValueVariant` after the dust on that settles.

Reply:
> radians

You mean operations like this right? `transformMatrix.operations.push_back(
          TransformOperation{TransformOperationType::Rotate, ValueUnit(toRadians(parameters, 0.0f), UnitType::Point), Zero, Zero})`

Yeah, I considered `UnitType::Point` like a regular `Float` as `ValueUnit` eventually will get replaced and to keep the PR small but we can add [Angle units](https://www.w3.org/TR/css-values-4/#angles) in `CSSValueVariant`.

> Replacing ValueUnit with CSSValueVariant 

For sure, i can see it is in `renderer/css`, can we import and start using it? We can also replace `ValueUnit` in `transformOrigin`, i guess a separate PR to do that would be better.